README.md

```diff
@@ -4,13 +4,17 @@
 [![Ansible Galaxy](http://img.shields.io/badge/galaxy-Datadog.datadog-660198.svg)](https://galaxy.ansible.com/Datadog/datadog/)
 [![Build Status](https://travis-ci.org/DataDog/ansible-datadog.svg?branch=master)](https://travis-ci.org/DataDog/ansible-datadog)
 
-Install and configure Datadog Agent & checks. Starting with version `2` of this role Datadog Agent version 6 is installed by default (instead of version 5).
+Install and configure Datadog Agent & checks.
+The version `3` of this role installs the Datadog Agent version 7 by default.
```

Comment:
Just realize that it's version 4 ! the role is already in version 3.3.0

Reply:
I just realized too! I'm going to change v3 -> v4 and v2 -> v3.